packages/admin-stories/src/admin/table/TableWithFilterBar.tsx

```diff
@@ -14,97 +14,161 @@ import { Typography } from "@material-ui/core";
 import { storiesOf } from "@storybook/react";
 import * as React from "react";
 
+import { generateRandomColors, generateRandomIntNumbers, generateRandomString } from "../helpers/DataGenerators";
+
+interface ExampleWithSelectProps {
+    colors: string[];
+}
+
+const ExampleWithSelect: React.FC<ExampleWithSelectProps> = ({ colors }) => {
+    const options = colors
+        .filter((color, index, colorsArray) => colorsArray.indexOf(color) == index)
```

Comment:
what are you trying to do with this filter?

Reply:
its a filter so i only have unique values in the select Options. Dont want to have many duplicated options in the select box :)

Reply:
ok, was not obvious to me :) please add a comment